src/content/reducers/index.js

```diff
@@ -6,8 +6,6 @@ import urlState from './url-state';
 import flameChart from './flame-chart';
 import timelineView from './timeline-view';
 
-import type { State, Reducer } from './types';
-
-const reducer: Reducer<State> = { app, profileView, summaryView, urlState, flameChart, timelineView };
+const reducer = { app, profileView, summaryView, urlState, flameChart, timelineView };
```

Comment:
My take here is that there is very little value to specify explicitely the types for such a dictionary; better let flow infer the types itself.